nfadeeva/assignment3/tests.py

```diff
@@ -0,0 +1,40 @@
+from assignment3.dictionary import Dictionary
+from assignment3.search_words import search_words, Grid
+import unittest
+
+
+class Test(unittest.TestCase):
+
+    def setUp(self):
+        self.d = Dictionary(['CAR', 'CAT', 'CARD', 'CART'])
+
+    # tests for the Dictionary class
+    def test_is_word_true(self):
```

Comment:
You can have more descriptive names, e.g. "test_word_is_in_dictionary" or "test_prefix_is_in_dictionary"